frontend/src/ui/Blocks/Series.tsx

```diff
@@ -193,46 +218,412 @@ type SeriesBlockContainerProps = {
     children: ReactNode;
 };
 
-const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = ({ title, children }) => (
-    <div css={{
-        marginTop: 24,
-        padding: 12,
-        ...title && { paddingTop: 0 },
-        backgroundColor: "var(--grey95)",
-        borderRadius: 10,
-    }}>
-        {/* This is a way to make this fancy title. It's not perfect, but it works fine. */}
-        {title && <div css={{ maxHeight: 50 }}>
-            <WithTooltip tooltip={title}>
-                <h2 css={{
-                    backgroundColor: "var(--accent-color)",
-                    color: "var(--accent-color-bw-contrast)",
-                    padding: "4px 12px",
-                    borderRadius: 10,
-                    transform: "translateY(-50%)",
-                    fontSize: 19,
-                    margin: "0 8px",
-                    lineHeight: 1.3,
-                    ...ellipsisOverflowCss(3),
-                    display: "-webkit-inline-box",
-                }}>{title}</h2>
-            </WithTooltip>
-        </div>}
-        {children}
-    </div>
+type View = "slider" | "gallery" | "list";
+
+type ViewContext = {
+    viewState: View;
+    setViewState: (view: View) => void;
+};
+
+const ViewContext = createContext<ViewContext>({
+    viewState: "gallery",
+    setViewState: () => {},
+});
+
+const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = ({ title, children }) => {
+    const [viewState, setViewState] = useState<View>("gallery");
+
+    return (
+        <ViewContext.Provider value={{ viewState, setViewState }}>
+            <div css={{
+                marginTop: 24,
+                padding: 12,
+                backgroundColor: "var(--grey95)",
+                borderRadius: 10,
+            }}>
+                <div css={{
+                    display: "flex",
+                    [`@media (max-width: ${VIDEO_GRID_BREAKPOINT}px)`]: {
+                        flexDirection: "column",
+                    },
+                }}>
+                    {title && <div css={{
+                        display: "inline-block",
+                        padding: "8px 12px",
+                    }}>
```

Comment:
Maybe in one line? Or... even better: can we remove this div and just add the CSS to the `h2`?